content/en/docs/concepts/services-networking/service.md

```diff
@@ -870,6 +879,40 @@ in those modified security groups.
 
 {{< /note >}}
 
+#### Other CLB annotations on TKE
```

Comment:
The first time you use an acronym like TKE, please spell it out.

Reply:
> The first time you use an acronym like TKE, please spell it out.

Thanks for your suggestions, I have modified the content according to your suggestions.